### PR TITLE
Bugfix/memory leaks

### PR DESCRIPTION
**Pull request by andrewkroh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,11 +32,10 @@ install-cfg:
 cover:
 	# gotestcover is needed to fetch coverage for multiple packages
 	go get github.com/pierrre/gotestcover
-	GOPATH=$(shell $(GODEP) path):$(GOPATH) $(GOPATH)/bin/gotestcover -coverprofile=profile.cov -covermode=count github.com/elastic/topbeat/...
+	GOPATH=$(shell $(GODEP) path):$(GOPATH) $(GOPATH)/bin/gotestcover -race -coverprofile=profile.cov -covermode=count github.com/elastic/topbeat/...
```
Comment (ruflin) [inline Makefile]: Is this a flag we should use across all beats?

**Comment by ruflin** [comment]
LGTM. Not sure why it added the .travis file in Godeps? Which version of Godeps are you using?

File: Makefile (hunk 1 above)
Comment (andrewkroh) [inline Makefile]: I recommend we do it for all beats during coverage and system tests.

**Comment by andrewkroh** [comment]
`godep version` shows `godep v18 (darwin/amd64/go1.5.1)`.

File: Makefile (hunk 1 above)
Comment (ruflin) [inline Makefile]: ok, I will add it to libbeat so with https://github.com/elastic/libbeat/pull/284 it will then be added to all beats.

Comment (andrewkroh) [inline Makefile]: Cool, I am reviewing that one now.

**Comment by ruflin** [comment]
Hm, same here. Just checked some other repos and the .travis.yml file is normally not added. Strange?

**Pull request merged**